python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -3,99 +3,278 @@
+                expected_deps.extend([
+                    'liblzma.so.5',
+                    'libpcre.so.1',
+                ])
+    # AIX specific deps.
+    elif platform_system == 'aix':
+        # This is the standard list of deps for AIX 5.3.
+        expected_deps = [
+            '/unix',
+            'libbsd.a',
+            'libc.a',
+            'libcrypt.a',
+            'libcrypto.a',
+            'libdl.a',
+            'libnsl.a',
+            'libpthreads.a',
+            'libpthreads_compat.a',
+            'libssl.a',
+            'libtli.a',
+            'libz.a',
+            ]
+        # sys.platform could be 'aix5', 'aix6' etc.
+        aix_version = int(sys.platform[-1])
+        if aix_version >= 7:
+            expected_deps.extend([
+                'libthread.a',
+            ])
+    # Solaris specific deps.
+    elif platform_system == 'sunos':
+        # This is the standard list of deps for a Solaris 10 build.
+        # For now, we include the major versions for Solaris libs.
+        expected_deps = [
+            'libaio.so.1',
+            'libc.so.1',
+            'libcrypt_i.so.1',
+            'libcrypto.so.0.9.7',
+            'libcrypto_extra.so.0.9.7',
+            'libdl.so.1',
+            'libdoor.so.1',
+            'libgen.so.1',
+            'libintl.so.1',
+            'libm.so.2',
+            'libmd.so.1',
+            'libmp.so.2',
+            'libnsl.so.1',
+            'librt.so.1',
+            'libscf.so.1',
+            'libsocket.so.1',
+            'libsqlite3.so.0',
+            'libssl.so.0.9.7',
+            'libssl_extra.so.0.9.7',
+            'libuutil.so.1',
+            'libz.so.1',
+            ]
+    # OS X specific deps.
+    elif platform_system == 'darwin':
+        # This is the minimum list of deps for OS X.
+        expected_deps = [
+            'ApplicationServices.framework/Versions/A/ApplicationServices',
+            'Carbon.framework/Versions/A/Carbon',
+            'CoreFoundation.framework/Versions/A/CoreFoundation',
+            'CoreServices.framework/Versions/A/CoreServices',
+            'SystemConfiguration.framework/Versions/A/SystemConfiguration',
+            'libSystem.B.dylib',
+            'libcrypto.0.9.8.dylib',
+            'libgcc_s.1.dylib',
+            'libssl.0.9.8.dylib',
+            'libz.1.dylib',
+            ]
+    return expected_deps
+
+
+def get_actual_deps():
+    """
+    Returns the list of actual deps obtained using a bash script helper.
+    """
+    try:
+        actual_deps = subprocess.check_output('./get_binaries_deps.sh',
+            shell=True).split()
+    except:
+        print 'Could not determine the deps for the new binaries.'
+    else:
+        return actual_deps
+
+
+def check_deps(expected_deps, actual_deps):
+    """
+    Returns unwanted deps for the python binary or the .so files in 'build/'.
+    """
+    # Check actual deps one by one to see if there is a substring of each of
```

Comment:
I'm a bit confused. In https://github.com/chevah/python-package/pull/13 you said: _[…] doctrings. Keep it simple. Don't add details about implementation. The docstring is there for the people using this method. It should state what it does and the format of the returned value_.

Besides, the comment doesn't actually describe what `expected_deps` and `actual_deps` are and what they should contain. Or there is something evading my attention?

So what should I do? Add what  what `expected_deps` and `actual_deps` are and what they should contain in the docstring? From PEP 257 I gather this is required. But then, it enforces things you haven't asked for, so I'm not sure.

What about the the 4-lines comment below that explains the logic checking the deps? Add that too to the docstring?. I have no opinion on that whatsoever and looking to the style guide and PEPs I'm not sure.